charts/astronomer/templates/houston/api/houston-jwt-certificate-secret.yaml

```diff
@@ -49,7 +49,6 @@ metadata:
     heritage: {{ .Release.Service }}
   annotations:
     # Indicate that we want to sync this to airflow namespaces
-    "kubed.appscode.com/sync": {{ (printf "platform-release=%s" .Release.Name) | quote }}
```

Comment:
We have to think about what we want to do here.

In order to make the new config-syncer compatible with previous iterations of the platform, the config-syncer is based on this very specific annotation, see following workflow:
- Get secrets in release namespace
- For each secret, check if it contains the annotation `kubed.appscode.com/sync`
- IF it does, it means the secret needs to be synchronized to the namespaces containing the label associated to the previous annotation (e.g. `kubed.appscode.com: platform-release=astronomer)

If you remove this annotation here, config-syncer + commander won't work anymore since it's looking for this annotation.

2 solutions in my opinion:
1. Keep this annotation as is, but it means we keep something related to kubed in the codebase.
2. Rename to annotation to somethhing like `astronomer.io/config-syncer/sync`

If we decide to go for nb 2, I need to submit a 1 line PR on Commander to change the annotation used, and release it again.

What do you think ?

Reply:
Agreed, I kept it now for the transition but let's move to `astronomer.io/config-syncer/sync` now

Reply:
Agreed, I kept it now for the transition but let's move to `astronomer.io/config-syncer/sync` now